bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -114,6 +116,10 @@
     protected final static String ENABLE_BOOKIE_FAILURE_TRACKING = "enableBookieFailureTracking";
     protected final static String BOOKIE_FAILURE_HISTORY_EXPIRATION_MS = "bookieFailureHistoryExpirationMSec";
 
+    // Netty
+    protected final static String NETTY_USE_POOLED_BUFFERS = "nettyUsePooledBuffers";
+    protected final static boolean DEFAULT_NETTY_USE_POOLED_BUFFERS = true;
```

Comment:
it is not a common practice to have the default value here. I'd prefer stick to current practice. If we are going to change the practice, let's change them all.